review_wof/__init__.py

```diff
@@ -1,19 +1,20 @@
 """review_wof.py - Build report of patches and reviews
 """
+import yaml
 from flask import Flask, render_template
 
+from .model import patch_reviewers_view, patch_summary_view, patches_view
 from .model.gerrit_patches import GerritPatches
-from .model import patches_view
-from .model import patch_summary_view
-from .model import patch_reviewers_view
-
 
 application = Flask(__name__)
 
 
 @application.route('/')
 def index():
-    patches = GerritPatches()
+    with open("config.yml", 'r') as ymlfile:
```

Comment:
I think an environment variable would be a more forward-looking way to store configuration (5 factor, etc.), also all this code belongs in the model layer, not here in the view layer

If we want to have something the represent a list of servers - we need a GerritServers class in the model layer, but I actually do not think that is a useful abstraction because dividing patches by server is only marginally useful, as teams can typically have work that spans multiple servers. What we should aim for instead, is to merge together information from multiple servers and in the long term - allow division by team from the UI.

Reply:
I need to read your comment few times to understand what you are aiming for :-)

Reply:
Have done that several times with no avail

Reply:
Can I assume a DataSource is a Git server. Valid values would be: Gerrit, GitHub, Gitlab.